graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/execution/QueryHandler.kt

```diff
@@ -54,9 +55,9 @@ open class SimpleQueryHandler(private val graphql: GraphQL, private val dataLoad
             graphql.executeAsync(input)
                 .await()
                 .toGraphQLResponse()
-        } catch (e: Exception) {
-            val graphQLError = SimpleKotlinGraphQLError(e)
-            GraphQLResponse(errors = listOf(graphQLError))
+        } catch (exception: Exception) {
+            val graphQLError: GraphQLKotlinError = if (exception is graphql.GraphQLError) exception.toGraphQLKotlinType() else GraphQLKotlinError(exception.message.orEmpty())
```

Comment:
if exception is `graphql.GraphQLError` there is no need to wrap it in the kotlin type -> I guess you are doing that so we avoid serialization of exception/throwable field?